README.md

```diff
@@ -416,9 +417,9 @@ for city in ["new york", "paris", "berlin"]:
 
 ### Batch Asynchronous Searches
 
-We do offer two ways to boost your searches thanks to `async` parameter.
- - Blocking - async=false - it's more compute intensive because the search would need to hold many connections. (default) 
-- Non-blocking - async=true - it's way to go for large amount of query submitted by batch  (recommended)
+We offer two ways to boost your searches thanks to the`async` parameter.
+ - Blocking - async=false - more compute intensive because the search needs to maintain many connections. (default) 
+- Non-blocking - async=true - the way to go for large batches of queries  (recommended)
```

Comment:
```suggestion
 - Blocking - `async=false` - more compute intensive because the search needs to maintain many connections. (default) 
- Non-blocking - `async=true` - the way to go for large batches of queries  (recommended)
```